paddle/fluid/operators/conv_op.cc

```diff
@@ -132,6 +132,11 @@ void Conv2DOpMaker::Make() {
             "(Tensor) The output tensor of convolution operator. "
             "The format of output tensor is also NCHW.")
       .Reuse("Input");
+  AddInput("ResidualData",
+           "(Tensor) Tensor with residual data "
+           "to which convolution output will be added."
+           "Used on with fuse_eltwise fusion.")
```

Comment:
Used "on with" fuse_eltwise fusion doesn't sound quite right to me.

Reply:
What is your suggestion?